shared/actions/people.js

```diff
@@ -109,13 +111,14 @@ const _onTabChange = (action: RouteTypes.SwitchTo) => {
   const list = I.List(action.payload.path)
   const root = list.first()
 
-  if (root !== peopleTab) {
+  if (root !== peopleTab && _wasOnPeopleTab) {
```

Comment:
something still not ideal: this only checks if the user was on the people tab, rather than actually looking at the people screen. We shouldn't call `markViewed` unless the user is navigating away from the root

Reply:
I'll fix this in another PR

Reply:
nevermind, it was a really small change so I fixed it here